<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright (c) Kitware Inc. -->
<!-- SPDX-License-Identifier: BSD-3-Clause -->
<ui version="4.0">
 <class>pqIconBrowser</class>
 <widget class="QDialog" name="pqIconBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browse icons</string>
  </property>
  <layout class="QGridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="pqSearchBox" name="searchBox" native="true">
       <property name="settingKey" stdset="0">
        <string notr="true">showAdvancedProperties</string>
       </property>
       <property name="advancedSearchEnabled" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterBox"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QListView" name="listView">
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>150</width>
       <height>80</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
     </property>
     <property name="itemAlignment">
       <enum>Qt::AlignBottom|Qt::AlignHCenter</enum>
     </property>
     <property name="selectionRectVisible">
       <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="importIcon">
       <property name="toolTip">
        <string>Import new icon</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pqWidgets/Icons/pqOpen.svg</normaloff>:/pqWidgets/Icons/pqOpen.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remove">
       <property name="toolTip">
        <string>Remove custom icon</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/QtWidgets/Icons/pqMinus.svg</normaloff>:/QtWidgets/Icons/pqMinus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="removeAll">
       <property name="toolTip">
        <string>Remove all custom icons</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/QtWidgets/Icons/pqDelete.svg</normaloff>:/QtWidgets/Icons/pqDelete.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqSearchBox</class>
   <extends>QWidget</extends>
   <header>pqSearchBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../ApplicationComponents/Resources/pqComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pqIconBrowser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pqIconBrowser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
